packages/vats/src/bridge.js

```diff
@@ -61,12 +63,14 @@ const prepareScopedManager = zone => {
         assert(inboundHandler, X`No inbound handler for ${bridgeId}`);
         return E(inboundHandler).fromBridge(obj);
       },
-      setHandler(newHandler) {
+      setHandler(newHandler, oldHandler) {
         // setHandler could probably be on a separate facet to separate it
         // from the toBridge and fromBridge powers, but it's easier to
-        // implement a set once check for the handler and pass a single
+        // implement an already set check for the handler and pass a single
         // object around.
-        !this.state.inboundHandler ||
+        // We do allow explicitly replacing the handler by requiring the old
+        // handler to also be provided.
+        this.state.inboundHandler === oldHandler ||
           Fail`Bridge handler already set for ${this.state.bridgeId}`;
```

Comment:
This error message will be misleading when attempting a failed replacement, but only slightly—I think keeping it is probably fine.

Reply:
Please remove this assertion: it doesn't provide any clear benefit, and just makes the API for replacement more clumsy.